cupyx/scipy/special/_stats_distributions.py

```diff
@@ -867,6 +867,8 @@ def btdtri(a, b, p, out=None):
     }
     dk = k + 1;
     dn = n;
+    if (y <= 0.0) return 0.0;
+    if (y >= 1.0) return 1.0;
```

Comment:
`scipy.special.nbdtri` currently uses `incbi` in [this file](https://github.com/scipy/scipy/blob/e60a60394d61fe0081fc5f05f68ee585e07e4406/scipy/special/cephes/incbi.c#L51), does not use `boost::math::ibeta_inv` like as `scipy.special.betaincinv`.